3rdparty/python/lockfiles/flake8.txt

```diff
@@ -4,10 +4,14 @@
 #
 # --- BEGIN PANTS LOCKFILE METADATA: DO NOT EDIT OR REMOVE ---
 # {
-#   "version": 1,
-#   "requirements_invalidation_digest": "815457a1baf6226c993e5468ccdf64c69fe7214d3d9237911c762733e0130526",
+#   "version": 2,
 #   "valid_for_interpreter_constraints": [
```

Comment:
This name has grown on me btw :) especially when paired with `generated_with_requirements`.

Reply:
The only reason why naming things is a hard problem in computer science is because people insist on being concise 😆